examples/pii_masking_example/llamaindex_agentic_fastapi/app_presidio.py

```
@@ -0,0 +1,382 @@
+                return ToolCallEvent(
+                    tool_calls=[
+                        ToolSelection(
+                            tool_id="fake",
+                            tool_name=tool_name,
+                            tool_kwargs=tool_args,
+                        )
+                    ]
+                )
+        except Exception as e:
+            (await ctx.get("current_reasoning", default=[])).append(
+                ObservationReasoningStep(
```

Ruff (0.8.2)

Undefined name `ObservationReasoningStep`

(F821)